conda/models/index_record.py

```diff
@@ -81,30 +56,100 @@ class Link(DictSafeMixin, Entity):
 EMPTY_LINK = Link(source='')
 
 
-class FeaturesField(ListField):
+def push_individual_feature(result_map, val):
+    k, v = translate_feature_str(val)
+    result_map[k] = v
+
+
+def _make_provides_features(track_features, instance):
```

Comment:
Can we talk a bit here about why we need a python feature or a numpy feature? I'm not seeing it. Why isn't your soft pinning approach sufficient?